library/Fission/Types.hs

```diff
@@ -36,42 +38,65 @@ import           Fission.Platform.Heroku.AddOn
 
 import qualified Fission.Web.Auth              as Auth
 import           Fission.Web.Server.Reflective
-import qualified Fission.Web.Types             as Web
 
-instance Has Web.Host cfg => MonadReflectiveServer (RIO cfg) where
-  getHost = Config.get
+-- | The top-level app type
+newtype Fission a = Fission { unFission :: RIO Config a }
+  deriving newtype ( Functor
+                   , Applicative
+                   , Monad
+                   , MonadIO
+                   , MonadUnliftIO
+                   , MonadReader Config
+                   , MonadThrow
+                   , MonadCatch
+                   , MonadMask
+                   )
+
+-- TODO move to Fisson.hs
+runFission :: MonadIO m => Config -> Fission a -> m a
+runFission cfg actions =
+  actions
+    |> unFission
```

Comment:
May be better named as `internalConfig` or something similar

Reply:
cute name though so this isnt an over my dead body :)

Reply:
Totally get what you're saying, but you're missing that the `newtype` also contains a complete `RIO`. i.e. it's not just the internal config, it's both a monad and the config.

`unFission` is literally "unwrap this from `Fission` to expose the underlying `RIO Config`".

Reply:
Ok, how about `unwrapFission` `internalRIOMonad` `getInternalRIOMonad`? My only concern with `unFission` is someone new coming in will wonder what is it to `unFission` something

Reply:
I mean, they'll also be confused by `unRIO`, `unReader`, `unWriter`, and so on. But sure, `innerRIO` or something can work.

Reply:
Sorry, to clarify (from DMs). That's not meant in frustration; more that they'll run into this. I've renamed it `unwrapFission` 👍